kroxylicious/src/main/java/io/kroxylicious/proxy/internal/util/MemoryRecordsHelper.java

```diff
@@ -9,31 +9,23 @@
 import org.apache.kafka.common.record.MemoryRecordsBuilder;
 import org.apache.kafka.common.record.RecordBatch;
 import org.apache.kafka.common.record.TimestampType;
-
-import io.netty.buffer.ByteBuf;
+import org.apache.kafka.common.utils.ByteBufferOutputStream;
 
 /**
  * This is introduces additional factory builder methods then {@link org.apache.kafka.common.record.MemoryRecords#builder} ones,
  * in order to use {@link ByteBufOutputStream}<br>
  *
  */
-public class NettyMemoryRecords {
+public class MemoryRecordsHelper {
```

Comment:
no longer Netty specific

Reply:
Is this class still adding enough value to justify being part of the API?

Reply:
I was planning to leave this internal and have it live alongside the two filters using it.